_docs_use-cases/use-case-consumption-get-repositoryattributes.md

```diff
@@ -48,8 +48,10 @@ The LocalAttributes can be specified using a complex query.
 
```

Comment:
The first sentence isn't easy to understand perhaps you can add a comma before 'you are the `owner` of and whose `shareInfo` is undefined.' in this PR.